pkg/url/url.go

```diff
@@ -396,6 +398,36 @@ func GetValidPortNumber(componentName string, portNumber int, portList []string)
 	return portNumber, fmt.Errorf("given port %d is not exposed on given component, available ports are: %s", portNumber, strings.Trim(strings.Replace(fmt.Sprint(componentPorts), " ", ",", -1), "[]"))
 }
 
+// GetValidExposedPortNumber checks if the given exposed port number is a valid port or not
+// if exposed port is not provided, a random 5 digit valid port will be generated and returned
+func GetValidExposedPortNumber(componentName string, exposedPort int) (int, error) {
```

Comment:
componentName is not used in the function. Also this function should be in the util package.

Reply:
Deleted `componentName` from the function parameter. 

IMO, the function is to valid the user provided exposed port, which should only be used for `odo url create`, so still should be in `url.go`

Reply:
OK